src/mscorlib/shared/System/Buffers/MemoryManager.cs

```diff
@@ -38,6 +33,21 @@ public abstract class MemoryManager<T> : IMemoryOwner<T>, IPinnable
         /// </summary>
         public abstract void Unpin();
 
+        /// <summary>
+        /// Returns a <see cref="System.Memory{T}"/> for the current <see cref="MemoryManager{T}"/> .
+        /// </summary>
+        /// <param name="length">The element count in the memory, starting at offset 0</param>
```

Comment:
nit: add periods at the end of each sentence within the xml comments (here and elsewhere)